File: cases/taint/javascript/model-summary-through-negative/Bridge.js
// The external-shaped boundary of dfb-template-model-summary-through: declared
// in its own fixture file, named as a boundary, and covered by a summary whose
// semantics the analysis must produce whether or not it reads these bodies.
// Both bodies are the identity function, so the bodies agree and only the two
// summaries disagree.
var Bridge = {
  pass: function pass(value) {
    return value;
  },
  hold: function hold(value) {
    return value;
  }
};

File: cases/taint/swift/model-summary-through-positive/Bridge.swift
enum Bridge {
    static func `pass`(_ value: String) -> String { value } // DFB-MODEL: module=DataFlowBenchTaintSwift; identity=DataFlowBenchTaintSwift.Bridge.`pass`(_ value: Swift.String) -> Swift.String; role=summary; binding=in:0,out:return
    static func hold(_ value: String) -> String { value } // DFB-MODEL: module=DataFlowBenchTaintSwift; identity=DataFlowBenchTaintSwift.Bridge.hold(_ value: Swift.String) -> Swift.String; role=summary; binding=no-flow
}
